README.md

```diff
@@ -76,9 +76,11 @@ Just beside your bot's icon, you will see your `CLIENT ID`. Copy the `CLIENT ID`
 
 Then replace the `YOUR_CLIENT_ID_HERE` with your bot's `CLIENT ID` in the link provided below. After that, you can use the link to add the bot to any existing or newly created Discord server (as long as you have the `Manage Server` permission).
 
-_https://discordapp.com/oauth2/authorize?client_id=YOUR_CLIENT_ID_HERE&scope=bot&permissions=8192_
+_https://discordapp.com/oauth2/authorize?client_id=YOUR_CLIENT_ID_HERE&scope=bot&permissions=268512272_
 
-**IMPORTANT NOTE :** When you click on the link and want to add the bot to a server, it'll ask for the `Manage Messages` permission. You have to grant the bot this permission or it will not work properly.
+**IMPORTANT NOTE :** When you click on the link and want to add the bot to a server, it'll ask for the following permissions. You must approve these permissions for the bot to work correctly:
+
+`Manage Roles`, `Manage Messages`, `Manage Channels`, `View Channels`, `Read Message History`, `Send Messages`
 
```

Comment:
Add another important note:
```
**IMPORTANT NOTE :** When setting up the roles within the server including the `suspended` role for the suspended feature for admins, verify the "bot role" that is **above** the suspended role, otherwise the bot will be unable to manage the `suspended` role.
```

Reply:
Have reworded this. :)